pipeline/metadata/5.3.yaml

```diff
@@ -735,6 +715,26 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - dmfg
+        - name: "Permissive Mode Scenarios"
```

Comment:
@pranavprakash20 can you add it to stage-3, stage-1 is exceeding suite limit. We may experience resource crunch if the number of test suites in each stage exceeds 5 to 6 numbers.

Reply:
noted, changes made